src/components/graphic.js

```diff
@@ -1,15 +1,11 @@
 import React from "react";
 import { GridContainer, Grid } from "@trussworks/react-uswds";
 
-const Graphic = ({ images, graphicHeadings }) => {
+const Graphic = ({}) => {
   return (
     <GridContainer>
       <Grid row>
-        {images.map((image) => (
-          <Grid col={3}>
-            <p>{image}</p>
-          </Grid>
-        ))}
+        <Grid></Grid>
```

Comment:
Is it necessary to have another <Grid> here? Can we get away with using <Grid row> only?

Reply:
You're commenting on outdated files somehow